### PR TITLE
Secondary nav

### PR DESCRIPTION
**Pull request by DanGodfrey**
image

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ module.exports = function(config) {
 
     // level of logging
     // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
-    logLevel: config.LOG_INFO,
+    logLevel: config.LOG_DEBUG,
```
Comment (jcart) [inline karma.conf.js]: Is this by design? Sounds like INFO is a more reasonable default.

File: src/common.module.js [hunk 2]
```diff
@@ -6,11 +6,13 @@
...
     'angular-svg-round-progressbar',
-    'ngAnimate',
+    'ngAnimate'
   ])
 
   .config(['$translateProvider', function ($translateProvider) {
     $translateProvider.preferredLanguage('en_CA');
     $translateProvider.useSanitizeValueStrategy('escapeParameters');
+
+    console.log($translateProvider.translations());
```
Comment (jcart) [inline src/common.module.js]: Please remove.

**Comment by jcart** [comment]
cc @6 I will pass the torch to you for a quick eyeball please

File: karma.conf.js (hunk 1 above)
Comment (DanGodfrey) [inline karma.conf.js]: Yup, karma is only for unit tests. This helps us debug failed tests more effectively. We may want to switch it back if the test suite grows to a size where the added logs slow it down but right now its pretty snappy.

File: src/common.module.js (hunk 2 above)
Comment (DanGodfrey) [inline src/common.module.js]: done.

**Pull request merged**